python/sglang/srt/openai_api/protocol.py

```diff
@@ -182,6 +182,7 @@ class CompletionResponseChoice(BaseModel):
     text: str
     logprobs: Optional[LogProbs] = None
     finish_reason: Optional[str] = None
+    stop_reason: Union[None, int, str] = None
```

Comment:
Can we have a better name for this? Can we call it something like `matched_stop_obj`, `matched_stop_reason`, or `matched_stop` so we can distinguish it better with `finish_reason`.